sale_timesheet_invoice_description/models/sale.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# © 2016 Carlos Dauden <carlos.dauden@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import api, models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = 'sale.order.line'
+
+    @api.multi
+    def _prepare_invoice_line(self, qty):
+        res = super(SaleOrderLine, self)._prepare_invoice_line(qty)
+        note = []
+        for line in self.env['account.analytic.line'].search(
```

Comment:
I'm thinking that this information should be linked here, but displayed/not displayed on invoice report instead of hardcoding the list on the description. That way, it would be much more versatile and with more formatting options. What do you think?